ngageoint/hootenanny-ui
# #453

How to test (can also refer to #453)

1) Open up a dataset that has reviews
2) Go to a review that has a blue and a red feature (I have had it work with just a red feature but it wasn't reliable)
3) Click a feature so the pallet appears
4) Click and hold on another feature (I usually click a road) and then drag/pan the screen until the radial menu disappears
5) Should now ~~be experiencing a phantom pallet~~ see the radial menu again.

The fix was setting the radialMenu to undefined, which was not happening if you were clicking, holding, and panning.
